device_backends/LogicalNameMapping/src/LNMBitRangeAccessPlugin.cc

```diff
@@ -243,7 +241,26 @@ namespace ChimeraTK::LNMBackend {
     }
     catch(std::out_of_range&) {
       throw ChimeraTK::logic_error("LogicalNameMappingBackend BitRangeAccessPlugin: " + info.getRegisterName() +
-          R"(: Missing parameter "numberOfBits".)");
+          R"(: Unparseable parameter "numberOfBits".)");
+      }
+    }
+
+    if(const auto it = parameters.find("dataInterpretationFractionalBits"); it != parameters.end()) {
```

Comment:
I find the parameter names `dataInterpretationFractionalBits` and `dataInterpretationSigned` a bit inconsistent with `numberOfBits`. All 3 together describe the data interpretation, and hence go into the fixed point converter. I would be fine with just using `fractionalBits` and `isSigned`. Alternatively one could put all 3 into one string parameter `dataInterpretation` or maybe better `dataRepresentation`, which then needs to be parsed. A possible string could be like `16.8U` (16 total bits, 8 fractional, unsigned) or `12.2S`(12 total bits, 2 fractional, signed). Please note that negative fractional bits might be necessary as well (e.g. `16.-2S`), at least the fixed point converter supports this and we are using it in different context.

Reply:
Agree, the names are coming from when you could configure the fixed point converter with a different number of bits, because I misunderstood the configuration. Should have renamed them when I removed that feature